libs/force.js

```diff
@@ -51,9 +80,119 @@ var force = (function () {
     // Reference to the Salesforce OAuth plugin
         oauthPlugin,
 
+    // Reference to the Salesforce Network plugin
+        networkPlugin,
+
     // Whether or not to use a CORS proxy. Defaults to false if app running in Cordova, in a VF page,
     // or using the Salesforce console. Can be overriden in init()
-        useProxy = (window.cordova || window.SfdcApp || window.sforce) ? false : true;
+        useProxy = (window.cordova || window.SfdcApp || window.sforce) ? false : true,
+
+    // Where or not to use cordova for oauth and network calls
+        useCordova = window.cordova,
+
+    // Testing only
+       requestHandler;
+        
+
+
+    /**
+     * Compute SalesforceMobileSDK for web app.
+     */
+    function computeWebAppSdkAgent(navigatorUserAgent) {
```

Comment:
Do we need this? If we're using native network stack, the user agent should get tagged on every REST request that goes out. Is this specifically for the browser XHR requests?